### ISSUE TITLE
term.ReadPassword in "golang.org/x/term" package does not work

### ISSUE THREAD

**Issue by the729** [edgelesssys/ego#188]
Example: 
```
package main
import (
    "fmt"
    "os"
    "syscall"
    "golang.org/x/term"
)
func main() {
    fmt.Print("Password: ")
    bytepw, err := term.ReadPassword(int(syscall.Stdin))
    if err != nil {
        os.Exit(1)
    }
    pass := string(bytepw)
    fmt.Printf("\nYou've entered: %q\n", pass)
}
```

When running inside SGX, the program hang at `term.ReadPassword`, not able to input password. 

Although you could argue that a password from terminal input is inherently insecure, it is still nice to have this feature for compatibility.

**Comment by thomasten** [comment]
Thanks for reporting. My initial investigation indicates that this is probably caused by a bug in a syscall implementation. Thus, we should fix it regardless of whether this particular use case is meaningful or not.

**Comment by thomasten** [comment]
fixed in v1.1.0

**Comment by the729** [comment]
Tested. Perfect!
